### DISCUSSION TITLE
Build a bootstraps.zip by build-bootstraps.sh

### DISCUSSION THREAD

**Discussion by ljd1996** [termux/termux-packages#19350] [Q&A] [answered]
According to https://github.com/termux/termux-packages/wiki/For-maintainers, for a custom package name, I want to compile a bootstraps.zip through build-bootstraps.sh. Which branch should I use to compile it?
<img width="327" alt="image" src="https://github.com/termux/termux-packages/assets/29947475/b684c68b-8457-44d6-b667-47eedfd40386">

**Comment by truboxl** [comment, marked as answer]
https://github.com/termux/termux-app/issues/3239#issuecomment-1420791883
